data_structures/stack/main.c

```diff
@@ -19,15 +19,16 @@ int main()
     //function for poping the element out
     void pop()
```

Comment:
Can you remove the nested functions like:
* push
* pop
* peep
* update
Because this isn't a standard in C